src/volatile_set.c

```diff
@@ -1,79 +1,1162 @@
 #include <string.h>
 #include "volatile_set.h"
+#include "rax.h"
 #include "zmalloc.h"
-#include "config.h"
 #include "endianconv.h"
 #include "serverassert.h"
+#include "hashtable.h"
+#include "server.h"
+#include <stdint.h>
```

Comment:
Why do you need to include server.h? It's good to avoid it if we can.

Let's not mix `#include <...>` and `#include "..."` here. Group all `<...>` includes together.

Reply:
sure